Source/FullScreenPokemon.d.ts

```diff
@@ -1052,7 +1052,9 @@ declare module FullScreenPokemon {
          * Which status to give, such as "Sleep", if applicable.
          */
         status?: string;
+    }
 
+    export interface IActiveMove extends IMoveSchema {
```

Comment:
I figure that since Dig and Teleport would use this it would be more accurate to just be an active move instead of HM. I can change it though